docs/sources/operators-guide/run-production-environment/production-tips/index.md

```diff
@@ -124,3 +124,29 @@ Running a dedicated Memcached cluster for each cache type is not required, but r
 
 We recommend securing the Grafana Mimir cluster.
 For more information about securing a Mimir cluster, refer to [Secure Grafana Mimir]({{< relref "../../secure/_index.md" >}}).
+
+## Network
+
+Most of the communication between Mimir components occurs over gRPC. The gRPC
+connection does not use any compression by default. This can be most noticeable in the connection between a
+distributor and an ingester.
```

Comment:
```suggestion
connection does not use any compression by default. This can be most noticeable is the connection between
distributors and an ingesters.
```